apps/microfrontend-platform-testing-app/src/app/test-pages/clear-outlet-then-send-message-test-page/clear-outlet-then-send-message-test-page.component.scss

```diff
@@ -0,0 +1,13 @@
+:host {
+  display: flex;
```

Comment:
The only child of this component is the `form`, so I suggest to use `display: grid` instead of the `flex` layout.

Reply:
We use a grid because it is easy to make the child occupy all the available space, both vertically and horizontally. We are using this pattern in many places. Of course, we could achieve a similar result with a flex layout, but then we would have to align the flex element with `flex: auto`. Setting the height of the form would also be an alternative.

Reply:
In the `form` you're using a grid, but what I meant is to use a grid in the `:host` as well, because the only child of the component is the `form`. So instead of 
```
display: flex;
flex-direction: column;
gap: 1em;
```
why not just use
```
display: grid;
```
?

Reply:
Of course, sorry, misread.